src/components/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { BrowserRouter, Route } from 'react-router-dom';
+
+import Home from './Home';
+import Contacts from './Contacts';
```

Comment:
Maybe rename 'Contacts' to something like ContactsPage. In the future you'll have a 'ContactsList' and a 'Contact' component. 'Contacts' naming might be misleading in this case.